README.md

```diff
@@ -11,7 +11,7 @@
 ## freecodecamp是什么？
 freecodecamp 是一个自由的开源编程社区，它致力于抹平教育差距，
 
-让像你我一样的平凡人也能享受到花费巨资才可能享受到的优质教育。
+让像你我一样的平凡人不用花费巨资也能享受到的优质教育。
```

Comment:
Well looks like we have to rework on the README.md file soon. And to be honest I don't see the meaning of this one line of change. It feels weird either way.